backend/src/main/scala/com/softwaremill/adopttapir/metrics/Metrics.scala

```diff
@@ -33,9 +53,9 @@ object Metrics {
         Builder.Sbt
       )
 
-    val names = fakeInstance.productElementNames.toArray.filterNot(excludedStarterDetailsFields.contains)
+    val names = fakeInstance.productElementNames.toArray.filterNot(excludedStarterDetailsFields.contains) :+ "operation"
     require(
-      names.length == fakeInstance.productElementNames.length - excludedStarterDetailsFields.size,
+      names.length == fakeInstance.productElementNames.length - excludedStarterDetailsFields.size + 1,
```

Comment:
`1` => `OperationNameOffset`? Perhaps  `fieldNames` or`lables` would be more descriptive, than `names`? If so `OperationFieldNameOffset` or similar should be used.

Reply:
I have introduced `additionalLabels` array that contains `operation` label (so far and it can be extended if needed later) and it can be used for both lines #56 and #58